src/pages/[l1_l2]/[...slug].tsx

```diff
@@ -173,20 +174,29 @@ const Topic: NextPage<TopicProps> = ({
 export default Topic;
 
 interface IParams extends ParsedUrlQuery {
-    slug: string;
+    l1_l2: string;
+    slug: string[];
```

Comment:
main change slug is array

l1_l2 = product , slug = ['a'] will result in route /product/a
l1_l2 = product , slug = ['a', 'b'] will result in route /product/a/b